### ISSUE TITLE
Oracle has changed authentication procedure on updates.oracle.com

### ISSUE THREAD

**Issue by farblos** [MarisElsins/getMOSPatch#18]
Seems that the updates.oracle.com URLs now also require authentication using the SSO HTML logon page (I guess previously this has required basic authentication only).

The symptom I see is that getMOSPatch rejects all my platform input since it cannot get any data from the HTTP server to verify it.  When I dumped the output of

```
  DownloadString("https://updates.oracle.com/Orion/SavedSearches/switch_to_simple");
```
I see a big blob looking like the standard Oracle SSO page (output partially truncated):

```
<html>
<script language="javascript" type="text/javascript">
function submitForm()
{
var hash = location.hash;
if (hash) {
if(hash.indexOf("#") == -1){
hash="#"+hash
}
document.myForm.action = document.myForm.action+hash;
}
document.myForm.submit();
}
</script><head><base target="_self"></head><body onLoad="submitForm()"><noscript><p>JavaScript is required. Enable JavaScript to use OAM Server.</p></noscript><form action="https://login.oracle.com/mysso/signon.jsp" method="post" name="myForm"><!------------ DO NOT REMOVE -------------><!----- loginform renderBrowserView ----->
```

**Comment by farblos** [comment]
Doo.  This is a dup of #17, of course.  Feel free to close it.

**Comment by MarisElsins** [comment]
Resolved in 86cad7e0dbf06afd074dc8c2208e9a5e67a2dd72
